src/sections/Home/index.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import WindowSection from "../../components/WindowSection";
+import * as S from './styles';
+import HomeVideo from '../../assets/video/videohome.mp4';
 
 export default function Home(){
     return (
         <WindowSection id='home'>
-            Home
+            <S.VideoHome loop autoPlay>
```

Comment:
Wrap the video on a Container that has max width 100vw and max height 100vh with overflow hidden and make the video behave differently on each device it's being rendered on

![image](https://user-images.githubusercontent.com/31679230/98044173-c48b8300-1e26-11eb-8777-bfad56fe843f.png)

Reply:
![image](https://user-images.githubusercontent.com/31679230/98044225-d79e5300-1e26-11eb-8e17-8a30be8e3af7.png)

Reply:
`muted` attribute is needed for it to autoplay on every device